spec/02-integration/14-tracing/01-instrumentations_spec.lua

```diff
@@ -401,7 +415,7 @@ for _, strategy in helpers.each_strategy() do
 
         local spans = cjson.decode(res)
         assert_has_spans("kong", spans, 1)
-        assert_has_spans("kong.dns", spans, 2)
+        assert_has_spans("kong.dns", spans, 3)
```

Comment:
This additional span that we expect comes from the fact that we're calling `named_srv` service that points to `localhost` - which needs to be looked up, right? The old one uses `helpers.mock_upstream_host` which is `127.0.0.1` which does not need to be looked up. Am I correct?

Reply:
100% correct